devcontainers/setup.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import json
+import typing
+import pathlib
+import argparse
+
+description = """
+This utility script helps you setting up development environment inside docker container
```

Comment:
This script looks nice, however, I am unsure if it's strictly required: We could simply ship *.devcontainer/arch-linux/devcontainer.json* and *.devcontainer/debian/devcontainer.json* files. These files are also automatically found by VSCode when opening the workspace, i.e. you can just select your preferred variant in a drop-down field.